### PR TITLE
Add volumeGetter to rkt.

### PR DESCRIPTION
**Pull request by yifan-gu**
This enables rkt get mounted volume informations.

/cc @dchen1107 @vmarmol @yujuhong

### CODE REVIEW

**Comment by yifan-gu** [comment]
Think we should actually move the `volume_manager.go` to subpackage?

File: pkg/kubelet/rkt/rkt_linux.go [hunk 1]
```diff
@@ -102,10 +101,16 @@ type runtime struct {
 	recorder            record.EventRecorder
 	prober              prober.Prober
 	readinessManager    *kubecontainer.ReadinessManager
+	volumeGetter        volumeGetter
 }
 
 var _ kubecontainer.Runtime = &runtime{}
 
+// TODO(yifan): Remove this when volumeManger is moved to separate package.
```
Comment (vmarmol) [inline pkg/kubelet/rkt/rkt_linux.go]: typo: Manager

**Comment by vmarmol** [comment]
LGTM

**Comment by vmarmol** [comment]
CI failing:

```
# github.com/GoogleCloudPlatform/kubernetes/pkg/kubelet/rkt
pkg/kubelet/rkt/rkt_linux.go:405: not enough arguments in call to r.volumeGetter.GetVolumes
pkg/kubelet/rkt/rkt_linux.go:578: undefined: volumeMap
pkg/kubelet/rkt/rkt_linux.go:578: too many arguments in call to r.preparePod
```

File: pkg/kubelet/rkt/rkt_linux.go [hunk 2]
```diff
@@ -396,6 +402,11 @@ func (r *runtime) makePodManifest(pod *api.Pod, volumeMap map[string]volume.Volu
 		})
 	}
 
+	volumeMap, ok := r.volumeGetter.GetVolumes()
```
Comment (yujuhong) [inline pkg/kubelet/rkt/rkt_linux.go]: Need to pass the podUID to GetVolumes

**Comment by yifan-gu** [comment]
Ehh, forget to run `install` again.

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**